lib/app/app.rb

```diff
@@ -90,6 +90,10 @@ class ConnectorsWebApp < Sinatra::Base
     )
   end
 
+  get '/describe' do
```

Comment:
snice this sends back metadata about the connector, I am wondering if this could not be added in what `GET /`  returns ?

Reply:
I think that would work 👍